quick-start/protractor.conf.js

```diff
@@ -14,20 +14,20 @@ exports.config = {
   capabilities: {
     'browserName': 'chrome',
     chromeOptions: {
-      args: ["--headless", "--disable-gpu", "--window-size=1920,1080" ]
-      //args: ["--window-size=1600,1000"]
+      args: ["--headless", "--disable-gpu", "--window-size=1920,1080"]
```

Comment:
are these numbers visible to e2e/specs/index.ts? possible to import them there and have them in just one place?

Reply:
Commented lines are removed